RepoUtilities/pre-commit.toInstall

```diff
@@ -0,0 +1 @@
+python3 RepoUtilities/pre-commit.py
```

Comment:
On Windows, it did not work for me, and prevented me from doing any commits, with this error: 
error: cannot spawn .git/hooks/pre-commit: No such file or directory

Adding a shebang like #!/bin/sh at the top of the pre-commit file ended up solving the issue. If it's something that works on mac as well, let's add it here.

Reply:
Seconding that for Windows.

Reply:
pushed the shebang, let me know if there's other issues